templates/plugin/shellhooks.yml.erb

```diff
@@ -0,0 +1,5 @@
+---
+# always enable HTTPS only
```

Comment:
If this is true, why allow users to pass the `listen_on` parameter and thus change this?

Reply:
My interpretation was that it works over http but allows *anyone* to run things on your proxy, hence using https only is recommended. My thinking was that if people know what they're doing, then we shouldn't prevent them from changing it

CC @lzap

Reply:
That is indeed what we have for every module: a sane default but nothing should be hardcoded.

Reply:
Can we change this message then? This message implies an absolute.

Reply:
Once we agree on the exact wording in https://github.com/theforeman/smart_proxy_shellhooks/pull/1 , then I'll pull the changes here.

Reply:
Done in https://github.com/theforeman/puppet-foreman_proxy/pull/651/commits/4979a02a18e4060b98c9be6bf71f6d928c701fd6